### ISSUE TITLE
Use equal-area world map projections (e.g. Gall-Peters)

### ISSUE THREAD

**Issue by igorwwwwwwwwwwwwwwwwwwww** [rails-girls-summer-of-code/summer-of-code#223]
Hi lovely RGSOC team!

I saw that you are using the common [Mercator](https://en.wikipedia.org/wiki/Mercator_projection) projection for the world maps ([example](http://railsgirlssummerofcode.org/blog/2016-06-24-more-teams-aye)).

[This clip](https://www.youtube.com/watch?v=vVX-PrBRtTY) from the show "The West Wing" makes a good case for some of the issues with this projection and suggests some alternatives, including equal-area projections.

I was wondering if you would like to consider using an alternate equal-area projection like [Gall-Peters](https://en.wikipedia.org/wiki/Gall%E2%80%93Peters_projection) or [Hobo-Dyer](https://en.wikipedia.org/wiki/Hobo%E2%80%93Dyer_projection) in the future.

Thank you! ✨

**Comment by anikalindtner** [comment]
yes, totally on board with that! 💖  thank you for pointing that out @igorwwwwwwwwwwwwwwwwwwww! @anasofiapinho maybe you can find a good picture of these maps and start using them for our materials?

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
I have tried to find some pretty ones to use and have had a hard time doing so. [The wikipedia one](https://en.wikipedia.org/wiki/Gall%E2%80%93Peters_projection#/media/File:Gall%E2%80%93Peters_projection_SW.jpg) looks quite nice, but has very prominent grid lines. Perhaps some doctoring could be done using some of the resources from [Wikimedia](https://commons.wikimedia.org/wiki/Category:Gall-Peters_projection).

**Comment by anasofiapinho** [comment]
Hey @igorwwwwwwwwwwwwwwwwwwww and @anikalindtner,

I know about Gall-Peters map and it's very interesting. However I am not sure we should use it because of the following reasons:
1. People are not used to it and it will be harder for them to find the locations
2. It's harder to find beautiful maps like you said before
3. I have image proportions I need to respect in order to publish on twitter and facebook and they are 2:1. This map has a bigger lenght than the other one and this means everything will be cut on previews and even blog posts wouldn't be so visible
4. The locations will look like this meaning that people wouldn't be able to see them, specially in Europe. I can't cut the map in height because I have locations on the bottom of the map. Making cuts in the map and zoom won't look good and it will be more work...
<img width="930" alt="screen shot 2016-06-24 at 13 26 41" src="https://cloud.githubusercontent.com/assets/3285560/16337802/06152948-3a10-11e6-953c-74b23687d85c.png">

**Comment by alicetragedy** [comment]
I'd like to reopen this discussion — I get it that it's “more work” to do this, but I think @igorwwwwwwwwwwwwwwwwwwww had a very good point here. Making places diverse and inclusive will always be more work, but I think given the type of program we are, we should give this a chance. The biggest challenge I see is in creating location maps as we've been doing in the past — @anikalindtner has been using google maps, which allows us to fairly easily pinpoint locations and get a good image out of it, and this work would now need to be done from scratch. If we really decide to go ahead with it, I can spend a little bit of time cleaning up some found images to use as a base for all of our maps. 
My aim would be to start using those by the first team announcement (mid-May).

**Comment by carpodaster** [comment]
I don't know if this relevant (and I haven't tried the python script mentioned in the accepted answer), but I found this: https://help.openstreetmap.org/questions/17935/view-osm-as-gall-peters-projection

Good side-effect: I'm not sure if it's ok to screenshot a Google map selection and re-use it elsewhere. Afaik, OSM's licensing allows that.

**Comment by alicetragedy** [comment]
wow, super cool @carpodaster — I was just thinking of this issue the other day and wondering whether we should look into this a bit or close the issue for the moment. 
I'll keep this open; maybe some contributors are interested in looking into it!
